lldb/tools/yaml2macho-core/CoreSpec.h

```diff
@@ -0,0 +1,55 @@
+//===-- CoreSpec.h --------------------------------------------------------===//
```

Comment:
Nit: new files shouldn't use this ASCII art anymore: https://discourse.llvm.org/t/is-c-in-header-files-still-relevant/83124
```suggestion
//===----------------------------------------------------------------------===//
```